components/ListView.js

```diff
@@ -67,8 +67,10 @@ class ListView extends React.Component {
   static propTypes = {
     autoHideHeader: React.PropTypes.bool,
     style: React.PropTypes.object,
-    data: React.PropTypes.array,
+    data: React.PropTypes.oneOfType([
+      React.PropTypes.object, React.PropTypes.array]),
```

Comment:
Certainly, the component [ListViewDataSource](https://facebook.github.io/react-native/docs/listviewdatasource.html#constructor) allows you to use the structure as an array or as an object (map). This means that you can pass a map structured in this way:
```javascript
{
  "postId1": {
     "id": "postId1",
     "title": "lorem ipsum",
     ....
  },
  "postId2": {
     "id": "postId2",
     "title": "lorem ipsum",
     ....
  },
  ...
}
```
Having the data in store with that structure, this means that you should not change them.
Actually it would also be given the option of passing the `getRowData method (dataBlob, sectionID, rowID);` which allows you to instruct the ListView to own structure.